internal/bin/sql/sqlgen.go

```diff
@@ -0,0 +1,371 @@
+// (c) Copyright IBM Corp. 2022
+//go:build go1.18
+
+package main
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	combinations "github.com/mxschmitt/golang-combinations"
+)
+
+const driverStmt = "driver.Stmt"
+
+var _conn_m = map[string]string{}
+var _stmt_m = map[string]string{}
+
+var arrayConn = []string{
+	"driver.Conn",
+	"driver.Execer",
+	"driver.ExecerContext",
+	"driver.Queryer",
+	"driver.QueryerContext",
+	"driver.ConnPrepareContext",
+	"driver.NamedValueChecker",
+}
+
+var arrayStmt = []string{
+	"driver.Stmt",
+	"driver.StmtExecContext",
+	"driver.StmtQueryContext",
+	"driver.NamedValueChecker",
+	"driver.ColumnConverter",
+}
+
+func inArray(el string, arr []string) bool {
+	for _, v := range arr {
+		if v == el {
+			return true
+		}
+	}
+
+	return false
+}
+
+func removeOnceFromArr(el string, arr []string) []string {
+	for k, v := range arr {
+		if v == el {
+			return append(arr[:k], arr[k+1:]...)
+		}
+	}
+	return arr
+}
+
+func main() {
+
+	fmt.Println("// (c) Copyright IBM Corp. 2022")
+	fmt.Println("// Code generated by sqlgen. DO NOT EDIT.")
+	fmt.Println("")
+	fmt.Println("package instana")
+	fmt.Println("import \"database/sql/driver\"")
+
+	arrayConnCopy := make([]string, len(arrayConn))
+	arrayStmtCopy := make([]string, len(arrayStmt))
+
+	copy(arrayStmtCopy, arrayStmt)
+	copy(arrayConnCopy, arrayConn)
+
+	gen("driver.Conn", "w_conn_", arrayConn)
```

Comment:
Is there a reason why we have a constant `driverStmt` but not one for `"driver.Comm"`? 
Maybe I will find out as I go through the rest of the code.

Reply:
Changed